frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -212,17 +211,6 @@ export const personsModalLogic = kea<personsModalLogicType<PersonModalParams>>({
         },
     }),
     listeners: ({ actions, values }) => ({
-        refreshCohort: () => {
```

Comment:
Looks like this wasn't doing anything useful as we remain on the person modal after saving a cohort, but would appreciate some confirmation.